packages/components/src/Drawer/Drawer.scss

```diff
@@ -50,15 +49,13 @@ $tc-action-bar-background-color: lighten($alto, 5%) !default;
 	width: 100%;
 }
 
-:global(.tc-with-drawer-wrapper) > .drawer-stacked +
-:global(.tc-with-drawer-wrapper)  > .drawer-stacked {
-	width: 98%;
-}
-
-:global(.tc-with-drawer-wrapper) > .drawer-stacked +
-:global(.tc-with-drawer-wrapper) > .drawer-stacked +
-:global(.tc-with-drawer-wrapper) > .drawer-stacked {
-	width: 96%;
+:global(.tc-with-drawer-wrapper) {
+	@for $i from 1 through 10 {
+		&:nth-child(#{$i}) > .drawer-stacked {
```

Comment:
this version of the selector is pretty well covered by most browsers https://caniuse.com/#feat=css-sel3